e2e-tests/src/test/java/com/wixpress/common/petri/e2e/FilterParametersConfigTest.java

```diff
@@ -0,0 +1,56 @@
+package com.wixpress.common.petri.e2e;
+
+import com.wixpress.petri.test.TestBuilders;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class FilterParametersConfigTest extends BaseTest {
+
+
+    private static Path original;
+    private static Path tempFile;
+
+    @BeforeClass
+    public static void startServers() throws Exception {
```

Comment:
It is similar, we started with trying to do both as part of the `SampleAppRunner` but found out that it is too late for us to reconfigure the yaml, the difference is `SampleAppRunner` changes the `labProps` file while the test changes the yaml file before the server is stating, otherwise it is too late for us.
So while they're similar in which files are being changed while tests are running in the first case the `SampleAppRunner` changed the file and in the other the test env needs to change the file before `SampleAppRunner` goes to work,
I think we should consider splitting the file manipulation from `SampleAppRunner` and moving it to the test env in a later task